src/languages/es.ts

```diff
@@ -594,6 +594,10 @@ export default {
         posted: 'Contabilizado',
         deleteReceipt: 'Eliminar recibo',
         routePending: 'Ruta pendiente...',
+        receiptAudit: 'Auditoría de recibos',
+        receiptVerified: 'Recibo verificado',
+        receiptNoIssuesFound: 'No se encontraron problemas',
+        receiptIssuesFound: (count: number) => `Se encontró ${count} problema(s)`,
```

Comment:
@Krishna2323 Did we verify if it is the correct translation? We can ask for the correct translations in slack channel.